### DISCUSSION TITLE
Limitazioni sul numero di chiamate concorrenti

### DISCUSSION THREAD

**Discussion by dmigliacci** [fattureincloud/api#451] [Q&A] [answered]
Buongiorno a tutti,
il nostro woocommerce utilizza il plugin esterno woofatture (che so non essere vostro) per inviare le fatture su FIC.

È possibile che se viene effettuato una chiusura massiva di boh 10 ordini contemporaneamente e quindi 10 processi di invio, si possa raggiungere il numero di chiamate concorrenti delle vostre api?

Riformulo, c'è un limite di chiamate concorrenti alle api per singolo account? Se si, quale è?

Grazie in anticipo!

**Comment by dmigliacci** [comment]
Direi che posso rispondermi da solo:

<img width="987" alt="Screenshot 2024-06-08 alle 12 15 56" src="https://github.com/fattureincloud/api/assets/43272717/1b2b956c-db57-4527-94a5-4f9a718d4497">

È possibile richiedere una verifica dei log del nostro account per verificare se si sono verificati errori 429 Too Many Requests?

**Reply by valmoz** [reply, marked as answer]
Buongiorno @dmigliacci 
certo, è possibile verificare se ci sono stati errori 429 sul suo account, ma ci servirebbe il suo company ID. Per questo motivo le suggerisco di scrivere al nostro supporto clienti allegando il link a questa discussione.

A disposizione
Mauro

**Reply by dmigliacci** [reply]
Grazie @valmoz provvedo subito.
